src/gui/accountsettings.cpp

```diff
@@ -840,8 +841,15 @@ void AccountSettings::slotAccountStateChanged()
         }
     } else {
         ui->_folderList->setItemsExpandable(false);
-        ui->addButton->setText(tr("Add Folder"));
-        ui->addButton->setToolTip(tr("You need to be connected to add a folder."));
+        ui->addButton->setEnabled(false);
+
+        if (_accountState->supportsSpaces()) {
+            ui->addButton->setText(tr("Add Space"));
```

Comment:
Any reason we have to change the button text at all?

Reply:
Default is folder, esp. when there is no account yet. So if you add an ocis account, it needs to be changed.